README.md

```diff
@@ -84,8 +84,4 @@ I would like to thank microverse for giving me this oppurtunity
 
 ## ❓ FAQ <a name="faq"></a>
 
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE.md) licensed.
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] I would suggest you add these lines again in your README and also add a working [MIT] link.